fhir/resources/DSTU2/procedure.py

```diff
@@ -103,8 +103,8 @@ class Procedure(DomainResource):
         description="Classification of the procedure.",
     )
 
-    code: ListType[fhirtypes.CodeableConceptType] = Field(
-        None,
+    code: fhirtypes.CodeableConceptType = Field(
+        ...,
         alias="code",
         title="List of `CodeableConcept` items (represented as `dict` in JSON).",
```

Comment:
Please change from //List of `CodeableConcept` items (represented as `dict` in JSON).//  to // Type `CodeableConcept` (represented as `dict` in JSON).//

Reply:
Oops, sorry for the oversight!